.trivyignore

```diff
@@ -0,0 +1 @@
+CVE-2022-29458
```

Comment:
Could you please add some details wrt this vulnerability? That way we can remove this later

Reply:
Done.